File: 2482-difference-between-ones-and-zeros-in-row-and-column/2482-difference-between-ones-and-zeros-in-row-and-column.cpp
class Solution:
    def onesMinusZeros(self, grid: List[List[int]]) -> List[List[int]]:
        n, m = len(grid), len(grid[0])
        onesRow, onesCol = [0]*n, [0]*m
        for i in range(n):
            for j in range(m):
                onesRow[i] += grid[i][j]
                onesCol[j] += grid[i][j]
        ans = [[0]*m for _ in range(n)]
        for i in range(n):
            for j in range(m):
                ans[i][j] = 2*onesRow[i] + 2*onesCol[j] - (n + m)
        return ans

File: 2482-difference-between-ones-and-zeros-in-row-and-column/2482-difference-between-ones-and-zeros-in-row-and-column.py
class Solution {
    public int[][] onesMinusZeros(int[][] grid) {
        int n = grid.length, m = grid[0].length;
        int[] onesRow = new int[n];
        int[] onesCol = new int[m];
        for(int i=0; i<n; i++){
            for(int j=0; j<m; j++){
                onesRow[i] += grid[i][j];
                onesCol[j] += grid[i][j];
            }
        }
        int[][] ans = new int[n][m];
        for(int i=0; i<n; i++){
            for(int j=0; j<m; j++){
                ans[i][j] = 2*onesRow[i] + 2*onesCol[j] - (n + m);
            }
        }
        return ans;
    }
}